templates/repo/issue/labels/edit_delete_label.tmpl

```diff
@@ -33,6 +33,13 @@
 				<div class="desc gt-ml-2 gt-mt-3 gt-hidden label-exclusive-warning">
 					{{svg "octicon-alert"}} {{.locale.Tr "repo.issues.label_exclusive_warning" | Safe}}
 				</div>
+				<br>
+			</div>
+			<div class="field label-is_archived-input-field">
+				<div class="ui checkbox">
+					<input class="label-is_archived-input" name="is_archived" type="checkbox">
+					<label>{{.locale.Tr "repo.issues.label_archive"}}</label>
```

Comment:
Since archived labels are a completely new concept, we need a help text, i.e. a tooltip or a static paragraph.
Perhaps something like `Archived labels will not be shown when searching for labels to apply to an issue. Issues with such labels will keep the label. Use them to get rid of labels that should not be used anymore without deleting existing information`

Reply:
Is this to be translated in other languages

Reply:
Yes.

Reply:
Rephrasing the text `Archived labels are excluded from the label search when applying labels to an issue. Existing labels on issues remain unaffected, allowing you to retire obsolete labels without losing relevant information.`

Reply:
I'd remove the `relevant`, but apart from that sounds good.

Reply:
<img width="663" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/d8aec949-999d-42c1-b540-7676bf43c473">